interfaces/feature-policy.idl

```diff
@@ -0,0 +1,3 @@
+partial interface HTMLIFrameElement {
```

Comment:
Wow, this is the least amount of IDL I've seen in any spec:
https://wicg.github.io/feature-policy/#integrations